core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -177,6 +177,12 @@ private TableProperties() {}
   public static final String AVRO_COMPRESSION_LEVEL_DEFAULT = null;
 
   public static final String ORC_STRIPE_SIZE_BYTES = "write.orc.stripe-size-bytes";
+
+  public static final String ORC_BLOOM_FILTER_COLUMNS = "write.orc.bloom.filter.columns";
+
+  public static final String ORC_BLOOM_FILTER_FPP = "write.orc.bloom.filter.fpp";
+  public static final double BLOOM_FILTER_FPP_DEFAULT = 0.05;
```

Comment:
nit: `BLOOM_FILTER_FPP_DEFAULT` => `ORC_BLOOM_FILTER_FPP_DEFAULT`?

Reply:
+1. Other configs are format specific for their defaults, and parquet doesn’t use this configuration which could cause confusion.